RecFCCeeCalorimeter/src/components/CaloTowerToolFCCee.h

```diff
@@ -198,6 +181,15 @@ class CaloTowerToolFCCee : public GaudiTool, virtual public ITowerToolThetaModul
   SegmentationType m_hcalFwdSegmentationType;
   /// decoder: only for barrel
   dd4hep::DDSegmentation::BitFieldCoder* m_decoder;
+  // Set of bools to record whether a proper segmentation was found
+  bool m_ecalBarrelSegmentationOK;
+  bool m_ecalEndcapSegmentationOK;
+  bool m_ecalFwdSegmentationOK;
+  bool m_hcalBarrelSegmentationOK;
+  bool m_hcalExtBarrelSegmentationOK;
+  bool m_hcalEndcapSegmentationOK;
+  bool m_hcalFwdSegmentationOK;
```

Comment:
I think we can get rid of these, but can also be done in a separate PR later

Reply:
The bool class members have now been removed, and the interface to retrievePhiThetaExtrema changed so that it returns StatusCode.  The pair to hold the phi and theta max values is now passed by reference.

Reply:
I still see them in the latest commit, will merge and fix (with the tests) in a new PR